jekyll/_cci2/configuration-reference.md

```diff
@@ -1391,6 +1391,139 @@ ignore | N | String, or List of Strings | Either a single tag specifier, or a li
 
 For more information, see the [Executing Workflows For a Git Tag]({{ site.baseurl }}/2.0/workflows/#executing-workflows-for-a-git-tag) section of the Workflows document.
 
+###### **`matrix`**
+A matrix job allows you to run a parameterized job multiple times with different
+arguments.
+
+Key | Required | Type | Description
+----|----------|------|------------
+parameters | Y | Map  | A map of parameter names to every value the job should be called with
+exclude | N | Map | A list of argument maps that should be excluded from the matrix
+alias | N | String | An alias for the matrix, usable from another job's `requires` stanza. Defaults to the name of the job being executed
+
+The following is a basic example of using matrix jobs. All features are
+explained in more detail below.
+
+```yaml
+workflows:
+  workflow:
+    jobs:
+      - build:
+        matrix:
+          parameters:
+            version: ["0.1", "0.2", "0.3"]
+            platform: ["macos", "windows", "linux"]
+```
+
+###### Excluding sets of parameters from a matrix
+{:.no_toc}
+Sometimes you may wish to run a job with every combination of arguments _except_
+some value or values. You can use an `exclude` stanza to achieve this:
+
+```yaml
+workflows:
+  workflow:
+    jobs:
+      - build:
+          matrix:
+            parameters:
+              a: [1, 2, 3]
+              b: [4, 5, 6]
+            exclude:
+              - a: 3
+                b: 5
+```
+
+The matrix above would result in 8 jobs being run: every combination of the
+parameters `a` and `b`, excluding `{a: 3, b: 5}`
+
+###### Requiring an entire matrix
+{:.no_toc}
+Another job can `require` an entire matrix (that is, every job in the matrix)
+using its `alias`. The `alias` defaults to the name of the job being invoked.
+```yaml
+workflows:
+  workflow:
+    jobs:
+      - deploy:
+          matrix:
+            parameters:
+              version: ["0.1", "0.2"]
+      - another-job:
+          requires:
+            - deploy
+```
+
+This means that `another-job` will require both `deploy-0.1` and `deploy-0.2` to
+finish before it runs. 
+
+You can also manually set the `alias` to a custom value. For example:
```

Comment:
Do you think this would help explain when the alias is useful, or just complicate things because the following example doesn't demonstrate any ambiguity being addressed?
```suggestion
You can also manually set the `alias` to a custom value, to clear up any ambiguity between job names.
For example:
```